zer0-os/zOS
# feat(redesign-sidekick): update left and right sidekick positioning and styling

### What does this do?
- updates left and right sidekick positioning and styling

### Why are we making this change?
- as per redesign requests

### How do I test this?
- run tests as usual
- run UI > check sidekick positioning and styles as per design
### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?
